web/sites/default/config/core.entity_view_display.taxonomy_term.coalition.full.yml

```diff
@@ -375,6 +386,17 @@ content:
     third_party_settings: {  }
     type: link
     region: content
+  neighborhoods_entity_view_1:
```

Comment:
@RichardDavies , I'm not sure the entity_view_1 is the one assigned by layout builder. Here in the coalitions "full" view mode it is showing both neighborhoods_entity_view_1 and neighborhoods_entity_view_map. Though, they are the same weight 🤔

Reply:
Ugh. So when I originally changed the view's machine name, it appears Drupal did not update the references to the original machine name in all of the various config files. I've manually cleaned them up now and removed references to the original machine name.